### ISSUE TITLE
Build for  v1.2.0  is broken

### ISSUE THREAD

**Issue by alxlenc** [zwacky/goodreads-profile-workflow#2]
Please let me first state that I am not a Javascript developer.

Node 12 has a problem with the generated build file dist/index.js. I have pasted here the GitHub Action error ouput:
```
Run zwacky/goodreads-profile-workflow@main
  with:
    goodreads_user_id: 117462291
    shelf: read
    template: - [$title]($url) by $author
    comment_tag_name: GOODREADS-READ-SHELF
    sort_by_fields: >user_rating,>average_rating
    max_books_count: 10
    readme_file_path: ./README.md
    output_only: false
/home/runner/work/_actions/zwacky/goodreads-profile-workflow/main/dist/index.js:6
`):(this.indentate=function(){return""},this.tagEndChar=">",this.newLine=""),this.options.supressEmptyNode?(this.buildTextNode=Xo,this.buildObjNode=Jo):(this.buildTextNode=Wr,this.buildObjNode=Yr),this.buildTextValNode=Wr,this.buildObjectNode=Yr}Fe.prototype.parse=function(e){return this.j2x(e,0).val};Fe.prototype.j2x=function(e,r){let t="",n="",s=Object.keys(e),i=s.length;for(let u=0;u<i;u++){let o=s[u];if(typeof e[o]!="undefined")if(e[o]===null)n+=this.indentate(r)+"<"+o+"/"+this.tagEndChar;else if(e[o]instanceof Date)n+=this.buildTextNode(e[o],o,"",r);else if(typeof e[o]!="object"){let c=this.isAttribute(o);c?t+=" "+c+'="'+this.options.attrValueProcessor(""+e[o])+'"':this.isCDATA(o)?e[this.options.textNodeName]?n+=this.replaceCDATAstr(e[this.options.textNodeName],e[o]):n+=this.repl
```
But, If I run the file localy using node 14 I don't see such error.

**Comment by alxlenc** [comment]
The problem appears when minimizing the Javascript build. I have removed the `minify` flag in the package.json in my fork just to test it and it's currently working.

**Comment by rsHalford** [comment]
I'm getting the same error output when I run the action

**Comment by zwacky** [comment]
thanks you both for the report.

I'd like to keep the minify option, since we also check in the `/dist/index.js` file.
that's why I'm about to put up a fix where lodash's orderBy is replaced by a non-3rd party lib utility function.

**Comment by zwacky** [comment]
just published [v1.2.1](https://github.com/zwacky/goodreads-profile-workflow/releases/tag/v1.2.1) that should address this issue. works for my workflow.
can you test this with your workflow properties?

**Comment by alxlenc** [comment]
Great.

 I have switched back to your branch on v1.2.1 and works great.
 
 I will close the issue.
 
Thanks a lot 😄
